Added to Primary Transmission and Viral Entry (#203)
greenelab/covid19-review: content/07.pathogenesis.md

#### Primary Transmission and Viral Entry

Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].

> ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145; @doi:10.1007/s00134-020-05985-9; @doi:10.1128/JVI.00127-20]. 

Author: Now that this sentence has been combined with the one above, I think this sentence can be deleted.

Reviewer: @anskelly Oh no, sorry I missed this! Do you want to make a PR to fix that and add yourself as an author? I just merged the text in! Let me know if you need help, I got my email back under control so I should see it!

Author: @rando2 Thank you! I think I just made another pull request to delete that sentence... I'll try to add myself as an author soon too! Thanks again for all your help -- it's much appreciated! :)

Reviewer: Thanks @anskelly! I realized I gave you a hard task to edit two files in the same PR -- if you use this link it should work: <https://github.com/anskelly/covid19-review/blob/patch-2/content/metadata.yaml>

#### Primary Transmission and Viral Entry

> Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052]. 

Reviewer: ```suggestion
Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
```

> ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145; @doi:10.1007/s00134-020-05985-9; @doi:10.1128/JVI.00127-20]. 

Author: I think it would be more succinct if we just combined this sentence with the first sentence, and add 2 references. Perhaps like this: 

Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].

> The virus invades host cells by using ACE2 for entry and the serine protease TMPRSS2 for S protein priming. 

Author: I think this sentence can be deleted, since the topic of protease priming is covered in more detail later in the paragraph.

> The virus invades host cells by using ACE2 for entry and the serine protease TMPRSS2 for S protein priming. 

Reviewer: ```suggestion
```

[What cells are primary infection sites for SARS-CoV-2?]
[What structural aspects allow for viral entry]?

Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052]. 

> 

Reviewer: ```suggestion
```
You don't need spaces between each sentence, just between paragraphs (but you did a great job of separating the sentences onto separate lines, which really helps for editing!)
You can modify this by navigating to the "Files changed" tab at the top, then clicking the three dots below the green "Review changes" button. From there you will be able to "Edit file" and then commit directly to your branch (anskelly:patch-1). This will automatically apply your changes to the current PR!

Author: Thank you very much! Sorry for the hassle... I just went ahead and deleted the extra spaces (I think)!

Reviewer 2: @rando2 @anskelly 
It looks great! I think my previous #199 can be ignored.

Author: @jinhui2 Thanks! I think you had some really great references! Like the JVI paper from Fang Li's group and the Wang et al. Cell paper! Maybe we can incorporate those?

Reviewer 2: @anskelly Yes. I think it's a great idea to incorporate those papers. Thank you!

Author: @rando2 Thank you very much for your help! I think it would be great if we could cite the following 2 references after the first sentence in this PR: [@doi:10.1016/j.cell.2020.03.045] and [@doi:10.1128/jvi.00127-20]. Should I go ahead and add them on my end? Is there a way to make sure @jinhui2 gets credit for these?

Reviewer: Hi @anskelly, thank you for the great question! If @jinhui2 sees this and is able to submit a [suggestion](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review) GitHub will actually record them as the author! I can do something tricky on the command line to try to make it look like @jinhui2 submitted it, but this way is a lot easier!

The S protein is a highly glycosylated trimer that requires two proteolytic cleavage events, leading to substantial conformational changes, to achieve viral fusion with the host cell membrane [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507]. 

Each protomer is composed of an S1 and an S2 subunit, which mediate receptor binding and viral fusion, respectively. 

The priming proteolytic events occur sequentially, first at the S1/S2 junction and then at the S2’ site, ultimately resulting in the shedding of the S1 subunit and transitioning of the S2 subunit to a more stable, fusion-conducive conformation. 

Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases it can use to cleave the S protein [@doi:10.1016/j.cell.2020.02.052]. 

> Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry in vitro [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9]. 

Reviewer: ```suggestion
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry _in vitro_ [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9]. 
```

Interestingly, SARS-CoV-2 S protein also contains a RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV [@doi:10.1038/s41586-020-2012-7]. 

> Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10.1016/S0092-8674(00)81771-7; @doi:10.1006/viro.1999.9716]. 

Reviewer: ```suggestion
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716]. 
```
DOIs with parentheses cause a bug -- if you want, you can convert these on the website shortdoi.org. But I didn't warn you about that so not your fault!

Recent crystal structures have shown that the receptor-binding domain (RBD) of SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507]. 

Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach. Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that from SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind SARS-CoV-2-RBD [@doi:10.1126/science.abb2507]. 

Promisingly though, sera from convalescent SARS patients did inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052]. 

> 

Reviewer: ```suggestion
```
